### PR TITLE
Order Cycle Management Reports

### PR DESCRIPTION
**Pull request by lin-d-hop**
Added a few things in this PR, expanding out the Order Cycle management Report section:
1) For the payment methods report I added a Balance field using the User Balance Calculator
2) I added a second report of delivery methods. This report is not yet complete, waiting on a field for frozen/chilled items.

How do I stop the merge commits (see below) from showing up on every PR? Do I actually have to cherry-pick each pull request? Seems a cumbersome way to handle this.

### CODE REVIEW

**Comment by lin-d-hop** [comment]
ok @pmackay told me i need to rebase after a fetch upstream. So hopefully that answers the last question. In the meantime do I need to cherry-pick out those commits into a new branch? :-/

**Comment by RohanM** [comment]
Rebasing would solve this problem. The workflow you could use might be something like this:

```bash
# Code code code
$ git fetch upstream
$ git rebase upstream/master

# Ready to create a PR
$ git push origin my-branch

# From here on, don't rebase or merge in upstream
```

Once you've pushed your branch to a remote server, it's bad juju to rebase it, since rebasing rewrites the history of commits, and it's very messy to reconcile them. At that point, when developing a PR, I think the best way to go is to stay on that version of master, and allow whoever's merging the PR to deal with the merge.

**Comment by RohanM** [comment]
In the meantime, probably the best way is to cherry-pick into a new branch and open a new PR.

File: lib/open_food_network/order_cycle_management_report.rb [hunk 1]
```diff
@@ -7,22 +9,45 @@ def initialize(user, params = {})
...
-         order.email,
-         ba.phone,
-         order.distributor.andand.name,
-         order.shipping_method.andand.name,
-         order.payments.first.andand.payment_method.andand.name,
-         order.payments.first.amount
-        ]
+      if is_payment_methods?
+        orders.map do |order|
+          ba = order.billing_address
```
Comment (RohanM) [inline lib/open_food_network/order_cycle_management_report.rb]: This method is getting really long. That makes it harder to scan, and it's a code smell that suggests that it's time to break it up into component parts. Consider extracting two methods, `payment_method_row(order)` and `delivery_row(order)`. Then this method can be shortened to something like:

```ruby
orders.map do |order|
  if is_payment_methods?
    payment_method_row order
  else
    delivery_row order
  end
end
```

**Comment by lin-d-hop** [comment]
Replaced by #317

**Pull request closed without merging**